src/containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader.tsx

```diff
@@ -21,34 +25,28 @@ export const AMMAccountHeader = (props: { data: AmmDataType }) => {
   const { t } = useTranslation()
   const localizedBalance1 = localizeBalance(balance, language)
   const localizedBalance2 = localizeBalance(balance2, language)
-  const tradingFeeTotal = 1000
   const localizedLPBalance = localizeNumber(lpBalance, language, {
     minimumFractionDigits: 0,
     maximumFractionDigits: 2,
   })
-  const localizedTradingFee = localizeNumber(
-    tradingFee / tradingFeeTotal,
-    language,
-    {
-      minimumFractionDigits: 0,
-      maximumFractionDigits: 3,
-    },
-  )
+  const localizedTradingFee = tradingFee
```

Comment:
Instead of needing this ternary, `formatTradingFee` should just return `undefined` if `tradingFee` is `undefined`. There's also a potential bug here if `tradingFee` is 0